CONTRIBUTING.md

```diff
@@ -51,11 +51,7 @@ Work-in-process tutorials might have content and code in other repos.
 
 ## Who works on the tutorials?
 
-- The Tendermint (All in Bits) DevX team develops the tutorial code and dependencies. Our team includes [@fadeev](https://github.com/fadeev), [@lubtd](https://github.com/lubtd), [@ilgooz](https://github.com/ilgooz), [@Pantani](https://github.com/Pantani), and other talented developers. Don't they do an amazing job?
-
-- The Tendermint (All in Bits) EcoDev Engineering team ([@toschdev](https://github.com/toschdev), [@barriebyron](https://github.com/barriebyron), [@alijnmerchant21](https://github.com/alijnmerchant21), and [@soniasingla](https://github.com/soniasingla)) manages the developer experience and owns the tutorial doc content.
-
-- The Tendermint (All in Bits) Design team owns the front end and publishing. Be sure to check with [@lovincyrus](https://github.com/lovincyrus) for details about the metadata for layout and publishing tutorials.
+Meet the people [behind the Cosmos SDK and contributors](https://github.com/cosmos/sdk-tutorials/graphs/contributors). 
```

Comment:
I am not really sure about removing it, would be great to have another view point here @toschdev? 🙌

Reply:
Now that the starport guide is out, we can remove it

Reply:
yes we can remove it, I agree, I wrote it in the first place :)